queries/water-z11.pgsql

```diff
@@ -35,6 +35,24 @@ FROM
         AND way_area::bigint > 25600 -- 4px
         AND mz_way12 && !bbox!
 
+    --
+    -- Water line geometries
+    --
+    UNION
+
+    SELECT name,
+           NULL AS area,
```

Comment:
Probably least confusing to leave it null rather than mixing areas and lengths, IMO.